BackendApartmentReservation/BackendApartmentReservation/Managers/IChecklistManager.cs

```diff
@@ -5,6 +5,7 @@ namespace BackendApartmentReservation.Managers
 {
     public interface IChecklistManager
     {
-        Task<DbEmployeeAmenitiesChecklist> CreateChecklistForEmployee(int employeeId, FlightReservationInfo flightReservationInfo, CarReservationInfo carReservationInfo);
+        Task<DbEmployeeAmenitiesChecklist> CreateChecklistForEmployee(int employeeId, FlightReservationInfo flightReservationInfo, CarReservationInfo carReservationInfo, LivingPlaceReservationInfo livingPlaceReservationInfo);

```

Comment:
Styling. Put every parameter in a different line

Reply:
Gonna fix this